bin/alpen-client/src/config.rs

```diff
@@ -0,0 +1,33 @@
+use alloy_primitives::B256;
+use strata_acct_types::AccountId;
+use strata_identifiers::CredRule;
+
+/// Chain specific config, that needs to remain constant on all nodes
+/// to ensure all stay on the same chain.
+#[derive(Debug, Clone)]
+pub(crate) struct AlpenEeParams {
+    /// Account id of current EE in OL
+    pub account_id: AccountId,
+
+    /// Genesis blockhash of execution chain
+    pub genesis_blockhash: B256,
+
+    /// Genesis stateroot of execution chain
+    pub genesis_stateroot: B256,
+}
```

Comment:
If you want these to remain constant, then make the fields non-`pub` or `pub(crate,super)` (or even the new fancy [`pub(in foo::bar)`](https://doc.rust-lang.org/reference/visibility-and-privacy.html#pubin-path-pubcrate-pubsuper-and-pubself)) and add only accessors with shared/non-exclusive/read-only references.